README.md

```diff
@@ -25,6 +25,8 @@ This wrapper has the following functions:
 * send_action(recipient_id, action)
 * send_raw(payload)
 * get_user_info(recipient_id)
+* send_get_started(gs_obj)
```

Comment:
Looks good! thanks!

I've tried it out and it works as intended.

Can we use the name `set` instead of `send` as these seem to be actions that sets the messenger profile and is not directed at any user or entity.

```
set_get_started(gs_obj)
set_persistent_menu(pm_obj)
```